src/notebooks/R Examples/Comparative Metagenomics.ipynb

```diff
@@ -1252,7 +1285,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "model.interpretation.plot(siamcat_obj, consens.thres = 0.8, fn.plot = 'interpretation_plot.pdf')"
+    "model.interpretation.plot(siamcat_obj, consens.thres = 0.5, fn.plot = 'interpretation_plot.pdf')"
```

Comment:
So, I still got `Error in model.interpretation.plot(siamcat_obj, consens.thres = 0.5, fn.plot = "interpretation_plot.pdf"): Some models do not contain any features!` (and an empty plot) here even with these changes... were you expecting this to now work?